### PR TITLE
Clean template artisan command

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Kernel extends ConsoleKernel
      * @var array
      */
     protected $commands = [
-        //
+        'App\Console\Commands\CleanTemplate'
```
Comment (bkuhl) [inline app/Console/Kernel.php]: Can we swap this out to use the `::class` constant?

**Comment by millidavids** [comment]
This feels... too structural. But then again it pretty much is just a CLI tool.

File: app/Console/Commands/CleanTemplate.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+class CleanTemplate extends Command
+{
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'clean:template';
```
Comment (bkuhl) [inline app/Console/Commands/CleanTemplate.php]: Can you update this to use the `$signature` protected property?  It'll also replace the `getOptions()` from below.

**Comment by bkuhl** [comment]
This would probably also be best abstracted into a composer package so that we can utilize it within both Laravel and Lumen.

File: app/Console/Commands/CleanTemplate.php (hunk 2 above)
Comment (millidavids) [inline app/Console/Commands/CleanTemplate.php]: @bkuhl Do you have a link to your presentation on the artisan commands? I remember you talking about this.

File: app/Console/Commands/CleanTemplate.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            $rows = file($fname);
+            $blacklist = "QuotesTableSeeder";
+
+            foreach($rows as $key => $row) {
+                if(preg_match("/($blacklist)/", $row)) {
+                    unset($rows[$key]);
+                }
+            }
+
+            file_put_contents($fname, implode('', $rows));
```
Comment (bkuhl) [inline app/Console/Commands/CleanTemplate.php]: Can you update these file methods to utilize `Illuminate/Filesystem/Filesystem`?

File: app/Console/Commands/CleanTemplate.php [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+     * @var string
+     */
+    protected $description = 'Cleans the example files out of the project.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function fire()
```
Comment (bkuhl) [inline app/Console/Commands/CleanTemplate.php]: Can we update this to be `handle()` to maintain consistency with L5.2 standards?

File: app/Console/Commands/CleanTemplate.php (hunk 2 above)
Comment (bkuhl) [inline app/Console/Commands/CleanTemplate.php]: Good thinking!  http://bkuhl.github.io/talk-artisan-console/assets/player/KeynoteDHTMLPlayer.html#4

**Comment by millidavids** [comment]
@bkuhl @jlaswell

**Comment by jlaswell** [comment]
:+1: Here. @bkuhl do you mind to take a look at this too?

**Comment by bkuhl** [comment]
:+1: I'd be in favor of doing the same thing in this CI that's being done in Laravel, where this command is run in CI and then tests are run again.

**Pull request merged**